ajax/libs/rpg-awesome/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "rpg-awesome",
+  "filename": "css/rpg-awesome.min.css",
+  "description": "A fantasy themed font and CSS toolkit.",
+  "keywords": [
+    "rpg",
+    "font",
+    "awesome",
+    "css",
+    "toolkit"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/nagoshiashumari/Rpg-Awesome.git"
+  },
+  "authors": [
+    "Daniela Howe",
+    "Ivan Montiel <idmontie@gmail.com>"
+  ],
+  "licenses": [
+    "MIT",
+    "OFL-1.1"
+  ],
+  "homepage": "https://nagoshiashumari.github.io/Rpg-Awesome/",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/nagoshiashumari/Rpg-Awesome.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "css/rpg-awesome*",
+          "fonts/rpgawesome-webfont*"
+        ]
+      }
+    ]
+  }
```

Comment:
Could we use npm auto-update, please?

https://www.npmjs.com/package/rpg-awesome

```
tree node_modules/rpg-awesome

node_modules/rpg-awesome
├── CONTRIBUTING.md
├── Gruntfile.js
├── LICENSE.md
├── README.md
├── bower.json
├── css
│   ├── rpg-awesome.css
│   ├── rpg-awesome.css.map
│   └── rpg-awesome.min.css
├── demo
│   └── index.html
├── fonts
│   ├── rpgawesome-webfont.eot
│   ├── rpgawesome-webfont.svg
│   ├── rpgawesome-webfont.ttf
│   └── rpgawesome-webfont.woff
├── package.js
├── package.json
├── rpg-awesome-tests.js
└── scss
    ├── _bordered-pulled.scss
    ├── _core.scss
    ├── _fixed-width.scss
    ├── _icons.scss
    ├── _larger.scss
    ├── _list.scss
    ├── _mixins.scss
    ├── _path.scss
    ├── _rotated-flipped.scss
    ├── _spinning.scss
    ├── _stacked.scss
    ├── _variables.scss
    └── rpg-awesome.scss

4 directories, 29 files
```